### PR TITLE
#3192 Landing page chain checks

### LINKED ISSUE #3192: Unify landing pages accross the chains
When I click on the landing popular collections which shows RMRK NFTs, it lands me on the BSX prefix of item.
I guess this should not happen? 
Either 
- redirect to RMRK item if we want to have it there 
- or
- show BSX popular collections?

https://user-images.githubusercontent.com/5887929/174479000-6a83ba46-482c-402c-a34c-b75668c5ec15.mov

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3192
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: utils/mixins/prefixMixin.ts [hunk 1]
```diff
@@ -20,4 +20,17 @@ export default class PrefixMixin extends Vue {
   get client(): string {
     return this.urlPrefix === 'rmrk' ? 'subsquid' : this.urlPrefix
   }
+
+  get prefixBuildOnText() {
+    switch (this.urlPrefix) {
+      case 'bsx':
+        return 'Basilisk'
```
Comment (vikiival) [inline utils/mixins/prefixMixin.ts:27]:
```suggestion
        return 'Basilisk NFT Pallet'
```

**Review by vikiival — DISMISSED** [review]

**Comment by vikiival** [comment]
Also I would extend middleware so `https://beta.kodadot.xyz/bsx` will go to `https://beta.kodadot.xyz`

**Comment by prachi00** [comment]
> Also I would extend middleware so `https://beta.kodadot.xyz/bsx` will go to `https://beta.kodadot.xyz`

I'll create a separate issue for this one

**Comment by vikiival** [comment]
Hello 👋 ,

currently @Jarsen136 is assigned to the code review.

Since he is also part @kodadot/code-review-guild I trust him to make a code review correctly. 💟

File: pages/index.vue [hunk 2]
```diff
@@ -1,17 +1,23 @@
 <template>
-  <Landing prefix="rmrk" />
+  <Landing :prefix="urlPrefix" :buildOn="buildOnText" />
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'nuxt-property-decorator'
+import { Component, mixins, Vue } from 'nuxt-property-decorator'
```
Comment (Jarsen136) [inline pages/index.vue:6]: How about removing this unused value `Vue`

**Review by Jarsen136 — COMMENTED** [review]
WFM

Let's fix this small stuff and it's good to go

**Comment by prachi00** [comment]
> WFM
> 
> Let's fix this small stuff and it's good to go

Done

**Comment by Jarsen136** [comment]
LGTM

pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 57.84 USD/KSM ~ 0.864 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x1663386d7abbbf6b7631ef62d776590ae425485e317bc806ae749abbb87680f6](https://kusama.subscan.io/extrinsic/0x1663386d7abbbf6b7631ef62d776590ae425485e317bc806ae749abbb87680f6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Review by Jarsen136 — APPROVED** [review]

**Pull request merged**